src/lib.rs

```diff
@@ -133,6 +133,9 @@ pub type ssize_t = isize;
 pub enum FILE {}
 pub enum fpos_t {} // TODO: fill this out with a struct
 
+#[cfg(not(any(dox, target_env = "msvc")))]
```

Comment:
Ideally this `#[cfg]` would get removed. Could the `dox` module get shimmed and is this just not defined on MSVC?